runtime/ftplugin/rust.vim

```diff
@@ -57,6 +57,50 @@ setlocal includeexpr=rust#IncludeExpr(v:fname)
 
 setlocal suffixesadd=.rs
 
+if !exists("*" .. expand("<SID>") .. "RustfmtConfigOptions")
+function s:RustfmtConfigOptions()
```

Comment:
I'd rather prefer to just call `rustfmt#RustfmtConfigOptions()` (you may need to change `s:RustfmtConfigOptions()` to `rustfmt#RustfmtConfigOptions()` inside `autoload/rustfmt.vim`.